code/game/turfs/simulated/floor/fancy_floor.dm

```diff
@@ -80,6 +80,11 @@
 	burnt = 1
 	update_icon()
 
+/turf/open/floor/carpet/carpetsymbol
+	icon_state = "carpetsymbol"
+
+/turf/open/floor/carpet/carpetsymbol2
+	icon_state = "carpetsymbol2"
```

Comment:
I originally added the SMOOTH_FALSE but I don't know anything about code so I was worried it wouldn't work. I'll add it now.